x-pack/plugins/ml/common/types/saved_objects.ts

```diff
@@ -27,3 +27,12 @@ export interface InitializeSavedObjectResponse {
   success: boolean;
   error?: any;
 }
+
+export interface DeleteJobCheckResponse {
+  [jobId: string]: DeleteJobPermission;
+}
+
+export interface DeleteJobPermission {
+  canDelete: boolean;
+  canUnTag: boolean;
```

Comment:
Looking for this word online, it seems `untag` is the generally accepted spelling, rather than `un-tag`. In which case, I think it makes sense to go with `canUntag` here.

Reply:
fixed https://github.com/elastic/kibana/pull/83871/commits/ceb41ead974a82cb78fa916b5466015f2f654b78